### DISCUSSION TITLE
Optimized, Individualized Spaced Repetition in Hierarchical Knowledge Structures

### DISCUSSION THREAD

**Discussion by jackliusr** [open-spaced-repetition/free-spaced-repetition-scheduler#28] [Ideas]
There is a new algorithm for Hierarchical Knowledge Structures, link is here  https://www.justinmath.com/individualized-spaced-repetition-in-hierarchical-knowledge-structures/. an implementation for anki will be a great help.

**Comment by L-M-Sherlock** [comment]
I knew that. But the algorithm is proprietary, we only know the outline instead of the implementation details.

By the way, it's also a huge work to build the encompassing graph in Anki.

**Reply by bliss4044** [reply]
What would make the encompassing graph so difficult? Are you referring to automatically creating an encompassing graph from a deck or the simple functionality of encompassing cards?

**Reply by L-M-Sherlock** [reply]
Due to privacy, developers don't have the right to have the access of content in Anki.
